### ISSUE TITLE
How many sendSync you can do before getting blocked?

### ISSUE THREAD

**Issue by hgc2002** [WHAnonymous/Chat-API#1154]
I mean, from normal usage point of view, doing so fast is too bad for wa, but, I don't know, perhaps doing once each 5 minute? 10 minutes? It can't be just once per day!

I've read that it got blocked right after two syncs. This can't be fine!

Any idea on this?

**Comment by mgp25** [comment]
@hgc2002 how many people do you add in your personal number daily?

**Comment by hgc2002** [comment]
Well, sometimes I've added 10 or 20 to make a group for a new activity, like playing games, go dancing, birthdays. Hard to tell. I'll try with a 5 minute delay. Any other idea?

**Comment by mgp25** [comment]
From the eyes of WA, you are abusing their system, not a good flow behaviour

**Comment by Raisel** [comment]
I worked with sync for a month and a half, 1 every 20 minutes, my number is not blocked but can not access more WA by api. I never sent messages to these numbers. I sent messages reminders, an agenda that builds on php, for my own number, always a 20 per week ...

Another thing, I tested with smaller periods of time, and under 15 minutes to sync function came to be blocked for a period of seconds. Conclusion, sinc 1 time per hour can be a good idea ... I am in need of help here to find out if it's my IP number was blocked, but had no success

Thank you
